src/scripts/fix-program-events-user/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "fix-program-events-users",
+  "version": "1.0.0",
+  "description": "Fix attributeCategoryOptions attribute for program events",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "test": "echo 'Error: no test specified' && exit 1"
+  },
+  "dependencies": {
+    "moment": "^2.15.2",
+    "request": "^2.34",
+    "request-promise": "^4.1.1",
+    "underscore": "^1.8.3",
+    "bluebird": "^3.5.0",
+    "basic-authorization-header": "^0.2.7",
+    "nconf": "^0.8.4"
+  },
+  "author": "Arnau Sanchez",
```

Comment:
@tokland I know we have the name of the developers in some scripts but we have decided to set the author to 'EyeSeeTea Team' (for instance in Data Dictionary). Can you please change it accordingly?